vendor/src/github.com/microsoft/hcsshim/layerutils.go

```diff
@@ -84,9 +85,14 @@ func layerPathsToDescriptors(parentLayerPaths []string) ([]WC_LAYER_DESCRIPTOR,
 	var layers []WC_LAYER_DESCRIPTOR
 
 	for i := 0; i < len(parentLayerPaths); i++ {
-		// Create a layer descriptor, using the folder path
+		// Create a layer descriptor, using the folder name
```

Comment:
@swernli i would love to have some kind of walkthrough for the windows graphdriver :S
I don't understand what ActivateLayer does and the hcsshim package has too few comments.
I'm trying to understand how Get() works and understand the implications for Docker of this name vs path change.